Source/References/StateHoldr-0.2.0.ts

```diff
@@ -250,6 +260,21 @@ module StateHoldr {
         */
 
         /**
+         * Clears the list of collectionKeys.
+         */
+        clearCollectionKeys(): void {
+            this.collectionKeys = [];
+        }
+
+        /**
+         * Retrieves and sets collectionKeys to the list saved in ItemHolder.
+         */
+        retrieveCollectionKeys(): void {
+            var keys: string[] = this.ItemsHolder.getItem("collectionKeys");
+            this.collectionKeys = typeof keys === "undefined" ? [] : keys;
+        }
```

Comment:
I also prefer to avoid exposing functions that return `void` and just have side effects whenever possible.